go/test/endtoend/vtgate/queries/subquery/subquery_test.go

```diff
@@ -41,8 +41,10 @@ func TestSubqueriesHasValues(t *testing.T) {
 	utils.AssertMatches(t, conn, `SELECT id2 FROM t1 WHERE id1 NOT IN (SELECT id1 FROM t1 WHERE id1 > 10) ORDER BY id2`, `[[INT64(1)] [INT64(2)] [INT64(3)] [INT64(4)] [INT64(5)] [INT64(6)]]`)
 }
 
+// Test only supported in >= v14.0.0
 func TestSubqueriesExists(t *testing.T) {
 	defer cluster.PanicHandler(t)
+	utils.SkipIfBinaryIsBelowVersion(t, 14, "vtgate")
```

Comment:
Yeah, this won't work. With the backport the issue is fixed in release-13, but not in 12. So, adding this line will actually skip the test for this branch too, so we might as well just delete this test, since it won't run on any CI.
We should actually have it set to 13 here, so that it works in the normal CI tests and also in upgrade tests when the binaries are from release-13

Reply:
Just saw that we added support for this query got added in v13, my bad :) I'll set it to 13, makes more sense if we support it.